test/test_serialization.py

```diff
@@ -876,8 +876,8 @@ def import_module(name, filename):
                 loaded = torch.load(checkpoint)
                 self.assertTrue(isinstance(loaded, module.Net))
                 if can_retrieve_source:
-                    self.assertEqual(len(w), 1)
-                    self.assertTrue(w[0].category, 'SourceChangeWarning')
+                    self.assertEqual(len(w), 2)
+                    self.assertTrue(w[1].category, 'SourceChangeWarning')
```

Comment:
Nit (check both categories)
And also `assertTrue` is wrong here, it should be `assertEqual`
```suggestion
                    self.assertEqual(w[0].category, FutureWarning)
                    self.assertEqual(w[1].category, 'SourceChangeWarning')
```